task10/task10/Program.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task10
+{
+    internal class Program
+    {
+        static void Main()
+        {
+            //задание 1
+            Console.WriteLine("Введите значение натурального b");
+            int b = int.Parse(Console.ReadLine());
+            Console.WriteLine("Массив:");
+            double[] array = new double[15];
+            int n = 1;
+            int i = 0;
+            array[0] = 1;
+
+            for (int p = 1; p < 15; p++)
+            {
+                n *= 2;
+                array[p] = n;
+            }
+
+            for (; i < array.Length; i++)
+            {
+                array[i] = array[i] - b;
+            }
+
+            PrintMassive(array);//задание 2
+
+            Console.WriteLine(string.Join("; ", ChangeSign(array)));//задание 3
+
+            SredArifm(array);//задание 4
+            int k = int.Parse(Console.ReadLine());
```

Comment:
Хотелось бы получить запрос не ввод числа _k_.
```suggestion
            Consol.WriteLIne("Введите число, остатки от деления на которое мы будем вычислять");
            int k = int.Parse(Console.ReadLine());
```